doc/source/whatsnew/v0.24.0.txt

```diff
@@ -704,7 +704,7 @@ Numeric
 Strings
 ^^^^^^^
 
--
+- Bug in :class:`StringMethods` where `str_contains()` was not filling missing values with given argument for na for a categorical Series (:issue:`22158`)
```

Comment:
can you use double backticks on Series, can you use the actual reference here, e.g. ``:func:`Series.str.contains` ``

Reply:
@jreback I've updated it, please let me know if there's anything else you'd like to change.

Reply:
@jreback Can you please merge this PR.